lms/static/js/learner_dashboard/views/unenroll_view.js

```diff
@@ -61,10 +80,28 @@ class UnenrollView extends Backbone.View {
     }
   }
 
+  startSubmit() {
+    this.$('.submit').prop('disabled', true);
+  }
+
   initialize(options) {
+    const view = this;
+
     this.urls = options.urls;
     this.isEdx = options.isEdx;
 
+    this.closeButtonSelector = '.unenroll-modal .close-modal';
+    this.$closeButton = $(this.closeButtonSelector);
+    this.modalId = `#${this.$el.attr('id')}`;
+    this.mainPageSelector = '#dashboard-main';
+
+    this.triggerSelector = '.action-unenroll';
+    $(this.triggerSelector).each(function setUpTrigger() {
+      const $trigger = $(this);
```

Comment:
Nitpick: instead of using ```this``` you can pass through an element from the function, which I think makes the code easier to read (https://api.jquery.com/each/).